main.go

```diff
@@ -113,11 +119,13 @@ func doPeriodicMode(es *elasticsearch.Elasticer, d *database.Databaser, client *
 
 	// Accept and handle messages sent out with the index.all and index.templates routing keys
 	client.AddConsumerMulti(
-		messaging.ReindexExchange,
-		"direct",
-		"templeton.periodic",
+		amqpExchangeName,
+		amqpExchangeType,
+		amqpQueueName,
```

Comment:
No, there is no use case for making the queue name configurable. I'll back those changes out.